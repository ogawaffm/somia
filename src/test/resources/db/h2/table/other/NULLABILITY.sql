DROP TABLE IF EXISTS TEST.NULLABILITY
/
CREATE TABLE TEST.NULLABILITY
(
    NULLABLE_COL     VARCHAR(10),
    NOT_NULLABLE_COL VARCHAR(10) NOT NULL
)