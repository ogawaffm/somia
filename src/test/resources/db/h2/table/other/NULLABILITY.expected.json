{
  "name": "NULLABILITY",
  "catalogName": "TEST",
  "schemaName": "TEST",
  "typeName": "BASE TABLE",
  "comment": "",
  "columns": [
    {
      "name": "NULLABLE_COL",
      "type": {
        "name": "CHARACTER VARYING",
        "typeNumber": 12,
        "className": "java.lang.String",
        "precision": 10,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 0,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 10,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    },
    {
      "name": "NOT_NULLABLE_COL",
      "type": {
        "name": "CHARACTER VARYING",
        "typeNumber": 12,
        "className": "java.lang.String",
        "precision": 10,
        "scale": 0,
        "isNullable": false,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 0,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 10,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    }
  ],
  "indexes": [],
  "collation": "",
  "primaryKey": null,
  "bestRowIdentifier": null
}